### DISCUSSION TITLE
CR Touch probe not working properly, trying to go through print bed

### DISCUSSION THREAD

**Discussion by Blitzkrieg916** [mriscoc/Ender3V2S1#1232] [General]
Hey, I just installed the Sprite Extruder Pro and hooked up the CR Touch I've been using forever.
When it probes the bed, it ignores it and tried to smash through it. 
When i place anything in the way of the probe (foam/paper/finger) it senses it and stops the nozzle from descending properly.
Any ideas what's going on?
I've tried a dozen different Z offset values.
I keep getting an error stating "Z position unknown, please Send Z to Home", which then causes the issue above.

I've linked to my reddit post with a video of the issue:
https://www.reddit.com/r/ender3v2/comments/1ack8mp/trouble_with_z_homing_sprite_extruder_pro_cr_touch/?utm_source=share&utm_medium=web2x&context=3

**Comment by mriscoc** [comment]
![image](https://github.com/mriscoc/Ender3V2S1/assets/2745567/1acf5242-a270-489c-aba4-500699ceb342)

https://github.com/mriscoc/Ender3V2S1/wiki/3D-BLTouch
